content/rvm/install.md

```diff
@@ -4,16 +4,27 @@ title: Installing RVM
 
 # Installing RVM
 
-## Quick (guided) Install
+RVM supports most UNIX like systems and Windows (with Cygwin). The basic requirements are `bash`, `curl`, `gpg2` and overall GNU version of tools - but RVM tries to autodetect it and install anything that is needed.
```

Comment:
https://www.howtogeek.com/265900/everything-you-can-do-with-windows-10s-new-bash-shell/